VoxelSniperCore/src/test/java/com/github/kevindagame/voxelsniper/SniperManagerTest.java

```diff
@@ -28,6 +30,9 @@ public void setUp() {
         Mockito.when(absplayer.hasPermission(Mockito.any(String.class))).thenReturn(true);
 
         var main = Mockito.mock(IVoxelsniper.class);
+        var config = Mockito.mock(VoxelSniperConfiguration.class);
+        Mockito.when(config.getDefaultBrush()).thenReturn("snipe");
+        Mockito.when(main.getVoxelSniperConfiguration()).thenReturn(config);
```

Comment:
Could we maybe instead of mocking this, use an actual config file?

Reply:
I don't think we should, since the config class depends directly on a filepath.